Admin/AdminBuilder.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Admin;
+
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+/**
+ * @author Christian Gripp <mail@core23.de>
+ */
+final class AdminBuilder implements AdminBuilderInterface
```

Comment:
That's a general decision from the team to close the API. If someone wants to extend stuff, he has to ask and explain why this point needs a public extension.